### PR TITLE
Footer2

### PR DESCRIPTION
**Pull request by emmaambr**
I've added a footer with information and social media links

footer
.

### CODE REVIEW

**Review by simonlj8 — APPROVED** [review]
Great work! I really like the animations on social media pictures. 
When i go to "Logga in", "Filmer" and "Contact" i see the content has moved to left side.

**Comment by emmaambr** [comment]
> 

The reason the content has moved to left is because I had to remove 'text-align: center' in globals.css.
We could resolve it before merging or create a new issue to resolve it later.

File: styles/globals.css [hunk 1]
```diff
@@ -7,7 +7,6 @@ body {
   background-color: black;
 
   align-items: center;
-  text-align: center;
```
Comment (mhema42) [inline styles/globals.css:10]: I saw that when text-align: center was removed from globals.css, we need to change in some .css files for individual text on different pages.

For example "kommande premiär" on startpage
image

Then there is the headline on movie, ticket and contact pages (some has no styling yet, so we can fix it later).
image

I don´t know if we should add text-align on the diferent pages now with this ticket or if we should fix it with some ticket for styling contactpage for example, or if we should add a new ticket for this fix?

**Review by mhema42 — APPROVED** [review]
Nice job Emma!
I think the footer looks very good and works on all pages, going back and forth :) 
Perhaps there could be a little more margin/padding on the left side to balance the text to the media pictures?
image

Also, i saw that when text-align: center was removed from global.css, some other pages was altered, i made a comment about that in "styles/globals.css" I think it is correct that it was removed and that we instead should align the text for the individual pages/text, but I don´t know if we should fix that now or if we should fix it in an other ticket?
Maybe we can discuss and decide this (tomorrow) before we merge this pull request.

Again good work Emma!

File: styles/globals.css (hunk 1 above)
Comment (emmaambr) [reply styles/globals.css:10]: My suggestion is that we resolve it by adding text-align in the individual css modules.

**Comment by emmaambr** [comment]
> other

Now that you mention it, margin to the left wouldn't be a bad idea. I'll fix it asap.

**Comment by emmaambr** [comment]
I've added some margin as suggested, pls check if OK before merging.

**Pull request merged**